aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -435,7 +435,20 @@
   "handlers": {
     "create": {
       "permissions": [
```

Comment:
What about SLR permissions? Do we need to include/handle it to be backward compatible?

ref: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAM.ServiceLinkedRoles.html